libs/components/molecules/src/lib/info-modal/info-modal.component.html

```diff
@@ -0,0 +1,4 @@
+<mat-label>{{ value }}</mat-label>
+<button mat-icon-button class="close-button" (click)="clsoeIconClick.emit()">
+  <mat-icon svgIcon="close"></mat-icon>
```

Comment:
Use existing material icons, i.e. `<mat-icon>close</mat-icon>`
Remove the svg you added